File: wallet/src/main/java/com/worldventures/wallet/analytics/settings/ResetPinSuccessAction.java
package com.worldventures.wallet.analytics.settings;

import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.core.service.analytics.Attribute;
import com.worldventures.janet.analytics.AnalyticsEvent;
import com.worldventures.wallet.analytics.WalletAnalyticsAction;

@AnalyticsEvent(action = "wallet:setup:settings:successfully reset your PIN",
                trackers = AdobeTracker.TRACKER_KEY)
public class ResetPinSuccessAction extends WalletAnalyticsAction {

   @Attribute("resetpin2") String resetPin = "1";
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/settings/ResetPinSuccessAction.kt
package com.worldventures.wallet.analytics.settings

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.Attribute
import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.wallet.analytics.WalletAnalyticsAction

@AnalyticsEvent(action = "wallet:setup:settings:successfully reset your PIN", trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class ResetPinSuccessAction : WalletAnalyticsAction() {

   @Attribute("resetpin2") internal var resetPin = "1"
}
